neurodamus/replay.py

```diff
@@ -60,13 +60,13 @@ def open_spike_file(self, filename, delay, population=None):
 
     @classmethod
     def _read_spikes_sonata(cls, filename, population):
-        import h5py
-        spikes_file = h5py.File(filename, "r")
-        # File should have been validated earlier
-        spikes = spikes_file.get("spikes/" + population)
-        if spikes is None:
+        import libsonata
+        spikes_file = libsonata.SpikeReader(filename)
+        if population not in spikes_file.get_population_names():
```

Comment:
Out of curiosity, `spikes_file.get(population)` works? (and returns `None` if population doesnt exist?)

Reply:
No, the only possible operations in the spikes_file are brackets and get_population_names()
https://github.com/BlueBrain/libsonata/blob/7a3983a4ac89535550075feafd5338a1693a1f5f/python/bindings.cpp#L1229